asab/web/authn/oauth.py

```diff
@@ -4,25 +4,31 @@
 import asab.web.rest
 
 
-async def oauthclient_middleware_factory(request, handler, url):
-	authorization = request.headers.get("Authorization", None)
-	if authorization is None:
+def oauthclient_middleware_factory(app, *args, url, **kwargs):
+
+	@aiohttp.web.middleware
+	async def oauthclient_middleware(request, handler):
+
+		authorization = request.headers.get(aiohttp.hdrs.AUTHORIZATION, None)
+		if authorization is None:
+			return asab.web.rest.json_response(request, {
```

Comment:
I propose to consider using: `raise aiohttp.web.HTTPUnauthorized()` instead of these custom JSON responses.